templates/places/hamburger.html

```diff
@@ -0,0 +1,24 @@
+{% extends 'base.html' %}
```

Comment:
It looks like the hamburger script was added to the homepage and you don't need this file.

Reply:
The script has been removed.